LayoutTests/platform/wpe/TestExpectations

```diff
@@ -825,14 +819,9 @@ webkit.org/b/127676 imported/w3c/web-platform-tests/xhr/send-redirect-to-cors.ht
 webkit.org/b/127676 imported/w3c/web-platform-tests/xhr/send-redirect-to-non-cors.htm [ Skip ]
 
 Bug(WPE) fast/css-grid-layout/flex-content-sized-columns-resize.html [ Timeout ]
-Bug(WPE) fast/css-intrinsic-dimensions/height-css-tables.html [ ImageOnlyFailure ]
-Bug(WPE) fast/css-intrinsic-dimensions/height-tables.html [ ImageOnlyFailure ]
 
 Bug(WPE) fast/dynamic/window-resize-scrollbars-test.html [ Timeout ]
 
-Bug(WPE) fast/parser/xml-declaration-missing-ending-mark.html [ Failure ]
-
-Bug(WPE) fast/shapes/shape-outside-floats/shape-outside-clip-path-selection.html [ Failure ]
```

Comment:
This one is also reported as failing in the EWS, but as a regular failure (not as image-only failure).